### PR TITLE
Fixes #27241 - duplicate content view CCV message is misleading

### PR DESCRIPTION
**Pull request by ianballou**
The mention of filters is a bit misleading here since it can be interpreted that the CCV undoes the filtering.

### CODE REVIEW

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/i18n/bastion_katello.pot [hunk 1]
```diff
@@ -2406,8 +2406,7 @@ msgstr ""
 
 #: app/assets/javascripts/bastion_katello/content-views/details/views/content-view-publish.html
 msgid ""
-"For the duplicate repositories, the packages in each Content View Version will combine regardless of the filters that \n"
-"        were applied to the individual Content View Version."
+"Duplicate repositories between the selected Content View Versions will merge, resulting in a Composite Content View with all packages that exist among the duplicates."
```
Comment (jlsherrill) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/i18n/bastion_katello.pot]: these files aren't meant to be updated manually

**Comment by ianballou** [comment]
Figured out how to update the `.pot` file.  Looks like quite a few updates were needed.  Shall I keep them in here?

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/i18n/bastion_katello.pot (hunk 1 above)
Comment (ianballou) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/i18n/bastion_katello.pot]: Figured out how to generate them.

Comment (jlsherrill) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/i18n/bastion_katello.pot]: oh no, i mean they aren't supposed to be updated at all as part of a pr.  All translation strings are updated around release time.  Apologies if i wasn't clear

Comment (ianballou) [reply engines/bastion_katello/app/assets/javascripts/bastion_katello/i18n/bastion_katello.pot]: Ah I gotcha, I'll remove it then

**Comment by jlsherrill** [comment]
[test katello]

**Comment by ianballou** [comment]
Not totally sure why the tests are failing here.

**Comment by jjeffers** [comment]
[test katello]

**Comment by jturel** [comment]
[test katello]

**Comment by jlsherrill** [comment]
merging based on @parhta's ack, thanks @ianballou !

**Pull request merged**